playground/backend/containers/python/Dockerfile

```diff
@@ -47,5 +47,9 @@ ENV BEAM_SDK="SDK_PYTHON"
 COPY --from=build /go/bin/server_python_backend /opt/playground/backend/
 COPY --from=build /go/src/playground/backend/configs /opt/playground/backend/configs/
 
+# Install Python Katas Utils
+COPY katas/log_elements.py /go/src/katas/
+ENV PYTHONPATH="/go/src/katas"
```

Comment:
Could we add a new folder to the python path instead of overriding it? 
${PYTHONPATH}:/go/src/katas

Reply:
Good point, added